js/huobi.js

```diff
@@ -1177,13 +1177,14 @@ module.exports = class huobi extends Exchange {
         const symbol = this.safeSymbol (marketId, market);
         const timestamp = this.safeInteger (order, 'created-at');
         const clientOrderId = this.safeString (order, 'client-order-id');
-        const amount = this.safeNumber (order, 'amount');
-        const filled = this.safeNumber2 (order, 'filled-amount', 'field-amount'); // typo in their API, filled amount
-        let price = this.safeNumber (order, 'price');
-        if (price === 0.0) {
+        const amount = this.safeString (order, 'amount');
+        const filled = this.safeString2 (order, 'filled-amount', 'field-amount'); // typo in their API, filled amount
+        let price = this.safeString (order, 'price');
+        const isPriceZero = Precise.stringEquals (price, '0');
```

Comment:
we can delete this code altogether, so just do 

```
const price = this.safeString (order, 'price');
```

this is because 0 is handled already in safeOrder2